bin/asinit

```diff
@@ -207,6 +212,36 @@ function ensureTsconfigJson() {
   console.log();
 }
 
+function ensureAsconfigJson() {
+  console.log("- Making sure that 'asconfig.json' is set up...");
+  if (!fs.existsSync(asconfigFile)) {
+    fs.writeFileSync(asconfigFile, JSON.stringify({
+      targets: {
+        debug: {
+          // -b build/untouched.wasm -t build/untouched.wat --sourceMap --debug
+          binaryFile: "build/untouched.wasm",
+          textFile: "build/untouched.wat",
+          sourceMap: true,
+          debug: true
+        },
+        release: {
+          // -b build/optimized.wasm -t build/optimized.wat --sourceMap --optimize
+          binaryFile: "build/optimized.wasm",
+          textFile: "build/optimized.wat",
+          sourceMap: true,
+          optimizeLevel: 3,
+          shrinkLevel: 1,
```

Comment:
Wondering if just `optimize: true` would work to inherit the defaults, in case these change.

Reply:
I believe it would. I'll update this configuration.